EXILED/Exiled.API/Features/Npc.cs

```diff
@@ -148,20 +152,33 @@ public static Npc Spawn(string name, RoleTypeId role, int id = 0, string userId
                 Log.Debug($"Ignore: {e}");
             }
 
-            if (RecyclablePlayerId.FreeIds.Contains(id))
+            if (!RecyclablePlayerId.FreeIds.Contains(id) && RecyclablePlayerId._autoIncrement >= id)
             {
-                RecyclablePlayerId.FreeIds.RemoveFromQueue(id);
-            }
-            else if (RecyclablePlayerId._autoIncrement >= id)
-            {
-                RecyclablePlayerId._autoIncrement = id = RecyclablePlayerId._autoIncrement + 1;
+                Log.Warn($"{Assembly.GetCallingAssembly().GetName().Name} tried to spawn an NPC with a duplicate PlayerID. Using auto-incremented ID instead to avoid issues.");
```

Comment:
```suggestion
                Log.Warn($"{Assembly.GetCallingAssembly().GetName().Name} tried to spawn an NPC with a duplicate PlayerID. Using auto-incremented ID instead to avoid an ID clash.");
```